src/containers/App/index.test.js

```diff
@@ -33,6 +59,21 @@ describe('<App />', () => {
     expect(wrapper.find(Footer).length).toBe(1);
   });
 
+  describe('routing', () => {
+    it('can navigate to kto form', () => {
+      const store = configureStore({}, memoryHistory);
+      const wrapper = mount(
+        <Provider store={store}>
+          <MemoryRouter keyLength={0} initialEntries={['/kto/ja/12345-abcsde']}>
+            <App {...props} />
+          </MemoryRouter>
+        </Provider>
+      );
+
+      expect(wrapper).toMatchSnapshot();
```

Comment:
Having a huge snapshot like this isn't very useful. Unless you use `render` instead of `mount`